update_ensembl_data.py

```diff
@@ -29,7 +41,7 @@ def write(self, path):
             yield f
         # TODO we can remove this once we've moved to using pre-commit and
         # Black, as these will be autoformatted.
-        subprocess.check_call(["black", "-q", path])
+        # subprocess.check_call(["black", "-q", path])
```

Comment:
I think we should keep the ``black`` call here - otherwise the code we're auto inserting will be horrible.

Reply:
Maybe add a comment here for developers for now saying "use ``pip install black`` if this command fails".

Reply:
Ok, I was having trouble with it but once I'm done with local testing I'll un-comment it.